README.md

```diff
@@ -21,7 +21,9 @@ Consider this file name:
 What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+Creating a migration file named create_artist. It creates tables for information
+for the MVC and the numbers at the beginning are time stamps of when the file
+was created.
 ```
```

Comment:
This is a migration file. It adds a new table to the database. The numbers are a timestamp, which helps my app "remember" which migrations have already run. This is important, because re-running a migration could overwrite my data.